front/app/containers/Admin/dashboard/components/filters/GroupFilter.tsx

```diff
@@ -33,7 +35,12 @@ const generateGroupOptions = (
   ];
 };
 
-const GroupFilter = ({ currentGroupFilter, onGroupFilter }: Props) => {
+const GroupFilter = ({
```

Comment:
Making this 'group filter' more reusable so that I can use only the placeholder (in the dashboard) or only the label (in the report builder settings of the demographics widget)